decidim-dev/lib/decidim/dev/test/rspec_support/component.rb

```diff
@@ -66,6 +66,6 @@
   end
 
   config.before do
-    Decidim.find_component_manifest(:dummy).reset_hooks!
+    # Decidim.find_component_manifest(:dummy).reset_hooks!
   end
```

Comment:
Should we reomve these?

Reply:
I have tried to revert this change in 350425fc0c . If the pipeline is failing, will be removed in a further commit.